### ISSUE TITLE
Credit & Questions

### ISSUE THREAD

**Issue by orwell96** [Andrey2470T/Advanced-Trains-Redo#1]
Hi
As I can see, you forked my project. You didn't change readme.txt either, just added a readme.md
1. Please remember to state in some places that this modpack was initially made by me and the people that are mentioned in the forum thread.
2. Did you do any changes to the core code (advtrains)? If not, and you won't do this in near future, consider this:
- Don't fork and copy my project. Instead publish a modpack on the forum that only contains the trains you (and mbb?) added. Then state on the forum that this modpack requires advtrains to run.
- Your advantage is that you don't need to worry about whether I update the advtrains core. The trains API will always stay backwards-compatible, so your trains will still work with future updates of advtrains.

**Comment by Andrey2470T** [comment]
No, core code i did not change. I did only added the trains by mbb and cabels and still crafting recipes for some trains. I purposely made this repository people not to install and compile all the trains that were added by mbb and they could download it with multiply of new trains free. That was all my intention for creating the repo.

**Comment by orwell96** [comment]
I've read your PM's. I'm sorry that im a bit busy at the moment.
I'll answer you soon.
